cfpb/sbl-frontend
# [Snapshot testing] Complete user profile form

Part 1 of #1062

## Changes

- Execution of snapshot tests controlled via ENV var
  - With snapshots: `yarn test:e2e:snapshot`
  - Without snapshots `yarn test:e2e`
- Implements snapshot tests for the `Complete user profile` form'

## How to test this PR

1. `yarn run test:e2e:snapshot`
2. Run the `complete-user-profile` tests
3. Notice the entry in the action log for `toHaveSnapshot`
    <img width="525" alt="Screenshot 2024-12-04 at 5 01 11 PM" src="https://github.com/user-attachments/assets/0d07d84f-4fc4-4834-866a-585b4e8b7838">

1. `yarn run test:e2e`
2. Run the `complete-user-profile` tests
3. Notice there is *no entry* in the action log for `toHaveSnapshot`
    <img width="526" alt="Screenshot 2024-12-04 at 5 00 02 PM" src="https://github.com/user-attachments/assets/ccd0d2fe-4546-49be-9fab-bf7c8c4b04a6">
